velox/connectors/hive/HiveConnectorSplit.h

```diff
@@ -49,6 +60,7 @@ struct HiveConnectorSplit : public connector::ConnectorSplit {
       dwio::common::FileFormat _fileFormat,
       uint64_t _start = 0,
       uint64_t _length = std::numeric_limits<uint64_t>::max(),
+      FileProperties _properties = FileProperties(),
```

Comment:
` FileProperties _properties = {}` should work. Same everywhere.

Reply:
Updated!